pkg/controller/directimagemigration/directimagemigration_controller.go

```diff
@@ -69,13 +69,9 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	// Watch for changes to MigClusters referenced by DirectImageMigrations
 	err = c.Watch(
 		&source.Kind{Type: &migapi.MigCluster{}},
-		&handler.EnqueueRequestsFromMapFunc{
-			ToRequests: handler.ToRequestsFunc(
-				func(a handler.MapObject) []reconcile.Request {
-					return migref.GetRequests(a, migapi.DirectImageMigration{})
-				}),
-		},
-		//		&ClusterPredicate{}
+		handler.EnqueueRequestsFromMapFunc(func(a client.Object) []reconcile.Request {
```

Comment:
@sseago I guess this is equivalent to the previous content here? It looks different but maybe equivalent.

Reply:
To the best of my knowledge it is equivalent. As with many other things, the API signatures have changed quite a bit between 0.1.z and 0.7.z. Where I've changed things like this it's because what we used to call doesn't exist any more, and this seems to have replaced it.